cmd/openshift-install/internal_integration_test.go

```diff
@@ -463,14 +483,23 @@ func isoContains(ts *testscript.TestScript, neg bool, args []string) {
 	isoPath, filePath := args[0], args[1]
 	isoPathAbs := filepath.Join(workDir, isoPath)
 
-	disk, err := diskfs.Open(isoPathAbs, diskfs.WithOpenMode(diskfs.ReadOnly))
-	ts.Check(err)
+	if strings.HasPrefix(isoPath, "agent.") || strings.HasPrefix(isoPath, "agentconfig.") {
```

Comment:
I'd really prefer to not have such kind of knowledge buried down deep in the custom command, wouldn't be possible to bubble it up at the script level? Even having two different commands would make it more readable/maintainable IMHO

Reply:
I can add a new function , say `existsInIso` but it will be doing the same work as in `isoContains`.